app/views/participants/fields/_financial.html.erb

```diff
@@ -5,7 +5,7 @@
 		<%= f.input :free_reduced_lunch_signed_up, :label => "Signed up" %>
 	<% end %>
 
-	<%= f.input :household_size, :collection => 1..20, :as => :select, :label => "Number of persons in household", :hint => "Include the student in this count." %>
+	<%= f.input :household_size, :collection => 1..20, :as => :select, :label => "Number of persons in household"%>
```

Comment:
Hey @mattharris5 , by migration you mean populating seeds.rb and running rake db:seed, to populate the database?